Jenkinsfile

```diff
@@ -14,19 +14,19 @@ pipeline {
         }
     }
     stages {
-        stage('Test and Deploy Extensions') {
+        stage('Build and Test') {
             steps {
                 checkout scm
                 dir('spring-filters-config') {
-                    git branch: 'fixMeteringIntegTests', changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/spring-filters-config.git'
+                    git branch: 'master', changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/spring-filters-config.git'
```

Comment:
I remember Pooja had a specific commit on the "fixMeteringTests" branch. Some tests failed without that commit on the "master" branch.

Reply:
@metacubed, I merged changes to set-env-metering-filter.sh into master. There is no reason to keep these changes on a branch.

Reply:
there are assertion steps in the IT test we commented out in the previous story, due to nurego API performance change, we think if we switch branch for testing, we may want to make sure the original IT test (non commented out version) still works at least locally. 
also, if we decides to switch branch for IT testing, the ReadMe file should update.

Reply:
I merged changes to set-env-metering-filter.sh into master because these changes are prerequisites for the tests to start.  These changes have nothing to do with the commented assertions and keeping them on a branch doesn't make sense.  It just creates confusion.

I agree with changing the README.

Reply:
agreed that set-env-metering-filter.sh can be merged into master, also I assume you will create a separate PR for the merging in spring-filters-config? btw, I think that change should be in first before this PR.

Reply:
@anny1099, this merge is already done.